src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/FilePathNormalizer.cs

```diff
@@ -23,6 +23,11 @@ public string NormalizeDirectory(string directoryFilePath)
         }
 
         public string Normalize(string filePath)
```

Comment:
There's no reason I could find that this class shouldn't be static (and therefore not have to be passed on the constructor), but I didn't want to touch everywhere in this otherwise small PR.